fission/src/aps/APS.ts

```diff
@@ -116,14 +125,38 @@ class APS {
         }
     }
 
-    static async convertAuthToken(code: string, codeVerifier: string) {
+    static async refreshAuthToken(refresh_token: string) {
+        let now
```

Comment:
This was added by me because I'm a *fantastic* developer. Could you remove the time check and instead replace with an async mutex?